Lib/test/test_warnings/__init__.py

```diff
@@ -899,6 +899,15 @@ def test_showwarning(self):
         self.module.showwarning(message, category, file_name, line_num,
                                 file_object, expected_file_line)
         self.assertEqual(expect, file_object.getvalue())
+        # Test that file is flushed after writing the warning.
+        raw = BytesIO()
+        file_object = TextIOWrapper(BufferedWriter(raw), 'utf-8',
+                                    newline='')
+        expect = self.module.formatwarning(message, category, file_name,
+                                           line_num)
+        self.module.showwarning(message, category, file_name, line_num,
+                                file_object)
+        self.assertEqual(raw.getvalue().decode('utf-8'), expect)
```

Comment:
```suggestion
        self.assertEqual(raw.getvalue().decode(), expect)
```

Reply:
First, I omitted it, but then I added it to make it consistent with passing the encoding to `TextIOWrapper` (which I think should stay). But I don’t feel as strong about this one and would be fine with omitting it.